WordPress/src/jetpack/AndroidManifest.xml

```diff
@@ -15,6 +16,46 @@
                     android:scheme="https" />
             </intent-filter>
         </activity>
+
+        <activity
+            android:name="org.wordpress.android.ui.AddQuickPressShortcutActivity"
+            android:label="@string/quickpress_shortcut_activity_label"
+            android:theme="@style/WordPress.NoActionBar"
+            android:enabled="true"
+            android:exported="true"
+            tools:replace="android:enabled"
+            tools:ignore="ManifestResource">
+            <intent-filter>
+                <action android:name="android.intent.action.CREATE_SHORTCUT" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
+        </activity>
```

Comment:
Why did we add AddQuickPressShortcutActivity in jetpack's manifest? It was already in `main/AndroidManifest.xml` but disabled for Jetpack. I remove `android:enabled` line from AddQuickPressShortcutActivity of `main/AndroidManifest.xml` file and it still worked.
Can we just remove `android:enabled` line instead of adding the same activity here?

Reply:
Good point.  Updated.  However, I thought it would be cleaner when removing it on WP, should make a note to add it to JP when removing.